.gitignore

```diff
@@ -21,18 +30,19 @@
 .byebug_history
 
 # Ignore database configuration
-config/database.yml
+# config/database.yml
```

Comment:
Do not add database.yml to the repository.

Reply:
Has been removed and added to .gitignore